ai2_internal/api.py

```diff
@@ -17,14 +16,18 @@ class Box(BaseModel):
     page: int
 
     @classmethod
-    def from_mmda(cls, box: mmda_ann.Box) -> Box:
+    def from_mmda(cls, box: mmda_ann.Box) -> "Box":
         return cls(
             left=box.l, top=box.t, width=box.w, height=box.h, page=box.page
         )
 
     def to_mmda(self) -> mmda_ann.Box:
         return mmda_ann.Box(
-            l=self.left, t=self.top, w=self.width, h=self.height, page=self.page
```

Comment:
@soldni it seems that this change is breaking the behavior of this function https://github.com/allenai/mmda/blob/main/mmda/types/box.py#L78

Do we need to update the Box class then?